src/include/net/netdevice.h

```diff
@@ -138,7 +138,8 @@ typedef struct net_device {
 #if defined(NET_NAMESPACE_ENABLED) && (NET_NAMESPACE_ENABLED == 1)
 	net_namespace_p net_ns;
 #endif
-
+	const struct wireless_ops *wireless_ops;  /* wireless operations */
```

Comment:
I don't see 'struct wireless_ops' declaration in this file. It can be broken if wireless is not enabled in the template